src/lib/mongoBackend/dbConstants.h

```diff
@@ -76,6 +76,12 @@
 #define CSUB_CONDITIONS         "conditions"
 #define CSUB_CONDITIONS_TYPE    "type"
 #define CSUB_CONDITIONS_VALUE   "value"
+#define CSUB_CONDITIONS_EXPR   "expression"
```

Comment:
The new `expression` field should be documented in https://github.com/telefonicaid/fiware-orion/blob/develop/doc/manuals/admin/database_model.md#csubs-collection document, eg:

expression: an expression used to evaluate if notifications has to be sent or not when updates come. It may be composed of the following fields: q, georel, geometry and/or coords (optional)

In addition, include expression in the document below the description.